packages/bierzo-wallet/src/routes/welcome/index.dom.spec.ts

```diff
@@ -0,0 +1,38 @@
+import TestUtils from 'react-dom/test-utils';
+import { Store } from 'redux';
+import { configureStore } from '../../store';
+import { RootState } from '../../store/reducers';
+import { expectRoute } from '../../utils/test/dom';
+import { PAYMENT_ROUTE } from '../paths';
+import { travelToWelcome } from './test/travelToWelcome';
+
+describe('The /welcome route', () => {
+  let store: Store<RootState>;
+  let welcomeDom: React.Component;
+
+  beforeEach(
+    async (): Promise<void> => {
+      store = configureStore();
+      welcomeDom = await travelToWelcome(store);
+    }
+  );
+
+  describe('has a button that', () => {
+    let button: React.ReactInstance;
+
+    beforeEach(
+      async (): Promise<void> => {
+        button = await TestUtils.findRenderedDOMComponentWithTag(welcomeDom, 'button');
+      }
+    );
+
+    it('exists', () => {
+      expect(TestUtils.isDOMComponent(button)).toBeTruthy();
```

Comment:
No need for this test, in case if `findRenderedDOMComponentWithTag` did not find any element it will throw an exception https://reactjs.org/docs/test-utils.html#findrendereddomcomponentwithtag

Reply:
Fixed by [2f36278](https://github.com/iov-one/ponferrada/pull/135/commits/2f36278ffdd8bd5dd484b0aa572b64e5a1670308).